ssr/src/page/post_view/mod.rs

```diff
@@ -61,7 +61,18 @@ pub fn ScrollingView<NV: Fn() -> NVR + Clone + 'static, NVR>(
         ..
     } = expect_context();
 
-    let muted = create_rw_signal(true);
+    let AudioState {
+        muted,
+        show_mute_icon,
+    } = expect_context();
+
+    set_timeout(
+        move || {
+            show_mute_icon.set(false);
+        },
+        Duration::from_secs(6),
+    );
+
```

Comment:
should we extract out this logic into separate component? We are duplicating this code everywhere

Reply:
Yes, we should. Pushing the update.